### ISSUE TITLE
Project Properties Keyboard Navigation getting stuck in Copyright Field again

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#33787]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

This was an issue previously (See #19134) which was fixed in 4.2 but is now back in 4.7.

STEPS:
1.  Open project properties. 
2. Arrow down the fields, Work Title, Composer, etc till you get to copyright.
3. Try arrowing further and you will see navigation is stuck.
4. Pressing Tab will take the user to the OK button and they may be unaware there were more possible fields, or unable to find a way back to them with keyboard navigation.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7.2

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
